public/app/plugins/panel/heatmap/ExemplarLabelEditor.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { StandardEditorProps } from '@grafana/data';
+import { InlineField, TagsInput } from '@grafana/ui';
+
+export const ExemplarLabelEditor = ({ value, onChange, context }: StandardEditorProps<string[]>) => {
+  const labels = context.options?.exemplars?.labels;
+  const setExemplarLabels = (ls?: string[]) => {
+    onChange(ls);
+  };
+  return (
+    <InlineField
+      label="Labels"
+      tooltip="Add exemplar labels to be displayed on exemplar tooltip. Note that the order of labels added is also important and the same order is displayed on the exemplar tooltip"
```

Comment:
Given the order of the tags in the editor determines the order displayed in the tooltip it might be nice to consider adding in the ability to easily reorder tags (via drag / drop w/e) to prevent some frustrating UX if you want to just change the first label (currently you must remove all existing labels and start from scratch in that case)


https://github.com/grafana/grafana/assets/22381771/f97e5bf5-38d4-41f4-b2eb-206c9973d6f8